sdk/quota/arm-quota/samples-dev/quotaGetSample.ts

```diff
@@ -12,8 +12,7 @@ import { DefaultAzureCredential } from "@azure/identity";
  */
 async function quotasGetRequestForCompute(): Promise<void> {
   const credential = new DefaultAzureCredential();
-  const subscriptionId = "00000000-0000-0000-0000-00000000000";
-  const client = new AzureQuotaExtensionAPI(credential, subscriptionId);
+  const client = new AzureQuotaExtensionAPI(credential);
```

Comment:
I don't think there are only 5 samples which are tenant-level ops for example https://github.com/Azure/azure-sdk-for-js/blob/39e0d0aeeae82a21f144f4d993c9e91d5d2f68a2/sdk/quota/arm-quota/src/api/groupQuotaLimits/operations.ts.

Could you go through this list to have a check https://github.com/Azure/azure-sdk-for-js/tree/39e0d0aeeae82a21f144f4d993c9e91d5d2f68a2/sdk/quota/arm-quota/src/api?